lib/Doctrine/DBAL/Migrations/Tools/Console/Command/AbstractCommand.php

```diff
@@ -117,20 +117,13 @@ protected function getMigrationConfiguration(InputInterface $input, OutputInterf
     }
 
     /**
-     * This method ensure that we stay compatible with symfony console 2.3 by using the deprecated dialog helper
-     * but use the ConfirmationQuestion when available.
-     *
-     * @param $question
+     * @param string $question
      * @param InputInterface $input
      * @param OutputInterface $output
      * @return mixed
      */
     protected function askConfirmation($question, InputInterface $input, OutputInterface $output)
     {
-        if (!$this->getHelperSet()->has('question')) {
-            return $this->getHelper('dialog')->askConfirmation($output, '<question>' . $question . '</question>', false);
-        }
-
```

Comment:
Our coverage report said we weren't covering this flow, can you point the test to me?